src/IO/H5/Python/File.cpp

```diff
@@ -51,6 +52,19 @@ void bind_h5file() {
            })
       .def("groups", +[](h5::H5File<h5::AccessType::ReadWrite>& f) {
         return std_vector_to_py_list<std::string>(f.groups());
-      });
+      })
+
+    .def("get_vol",
+         +[](const h5::H5File<h5::AccessType::ReadWrite>& f,
+             const std::string path) {
+           const auto& vol_file = f.get<h5::VolumeData>(path);
+             return &vol_file;
```

Comment:
The indentation here looks a little funny :)

Reply:
`clang-format` ted `:thumbsup:`